"use strict"; //para manejar errrores

try {
  x = 10;
} catch (error) {
  console.log(error);
}
console.log("continuamos");
